### DISCUSSION TITLE
Husky and Markdownlint for mdn/content! (Please execute `yarn install` locally)

### DISCUSSION THREAD

**Discussion by teoli2003** [mdn/mdn-community#230] [Announcements]
Hi, all! 🎉 

As part of our goal to improve the quality of MDN, we have added tests and bots to help us maintain a good MarkDown quality.

Checks have been running 🏃‍♀️ for a few weeks inside each PR: the Markdownlint linter flags problems it found in your changes. But you have nothing to do, as a bot 🤖  fixes them each morning.

**As some people (like me!) like to deliver a clean sheet, we now have added [Husky](https://typicode.github.io/husky/) and [lint-staged](https://github.com/okonet/lint-staged). They run pre-commits locally. So each time you commit some work, the huskies 🐕  run and fix Markdownlint problems for you.**

⚠️ _For this to work, you must re-run `yarn install` once in your (up-to-date) local installation._

You don't need to do anything if you are contributing via the GitHub UI. The warnings flagged are still automatically fixed the next day.

🍾 I want to give a big shout-out to the volunteers without whom this wouldn't have happened: @nschonni, @OnkarRuikar (the bot-master), @josh-cena, but also to @wbamberg and @elchi3 from [Open Web Docs](https://openwebdocs.org) that have championed this, as well as a big thank you to @caugner, @Rumyra and @schalkneethling from [Mozilla](http://www.mozilla.org) for reviews, help, and advice (and bug fixes!). 

cc/ @mdn/localization-team-leads

**Comment by OnkarRuikar** [comment]
Without the quick reviews from @teoli2003 and @caugner this wouldn't have become possible. It took a lot of PRs to tame the beast.

Looking forward to more automation:
- Prettier https://github.com/orgs/mdn/discussions/158
- Synchronization with browser compatibility data (BCD)  https://github.com/mdn/content/pull/20370
